packages/components/src/components/select/select.stories.ts

```diff
@@ -0,0 +1,427 @@
+ */
+
+export const DisabledMultiple = {
+  name: 'Disabled x Multiple',
+  parameters: {
+    controls: {
+      include: []
+    }
+  },
+  render: () => {
+    return html`<div class="h-[340px] w-full">
+      ${generateTemplate({
+        options: {
+          classes: 'w-full [&>tbody>tr>td]:w-[50%]'
+        },
+        axis: {
+          y: {
+            type: 'attribute',
+            name: 'useTags',
+            values: [false, true]
+          },
+          x: {
+            type: 'attribute',
+            name: 'disabled',
+            values: [false, true]
+          }
+        },
+        constants: [
+          clearableConstant,
+          multipleConstant,
+          fiveOptionsConstant,
+          {
+            type: 'attribute',
+            name: 'value',
+            value: 'option-1 option-2 option-3 option-4'
+          }
+        ],
+        args: null
+      })}
+    </div>`;
+  }
+};
+
+/**
+ * Shows available slots. The `label` and `help-text` slots will overwrite their corresponding attributes.
+ */
+
+export const Slots = {
+  parameters: {
+    controls: {
+      include: []
+    }
+  },
+  render: (args: any) => {
+    return html`
+      ${['default', 'label', 'clear-icon', 'expand-icon', 'help-text'].map(slot =>
+        generateTemplate({
+          axis: {
+            x: {
+              type: 'slot',
+              name: slot,
+              title: 'slot=...',
+              values: [
+                {
+                  value:
+                    slot === 'default'
+                      ? `<div class="slot slot--border slot--background h-8 w-full"></div>`
+                      : `<div slot='${slot}' class="slot slot--border slot--background h-6 ${
+                          slot === 'label' || slot === 'help-text' ? 'w-20' : 'w-6'
+                        }"></div>`,
+                  title: slot
+                }
+              ]
+            }
+          },
+          constants: [
+            { type: 'template', name: 'width', value: '<div style="width: 300px">%TEMPLATE%</div>' },
+            clearableConstant,
+            labelConstant,
+            helpTextConstant,
+            twoOptionsConstant
+          ],
+          args
+        })
+      )}
+    `;
+  }
+};
+
+/**
+ * Use the `form-control`,
+  `form-control-label`,
+  `form-control-input`,
+  `form-control-help-text`,
+  `combobox`,
+  `display-input`,
+  `listbox`,
+  `tags`,
+  `tag`,
+  `tag__base`,
+  `tag__content`,
+  `tag__remove-button`,
+  `tag__remove-button__base`,
+  `clear-button`, and
+  `expand-icon` part selectors to customize the select component.
+ */
+
+const partsArr = [
+  'form-control',
+  'form-control-label',
+  'form-control-input',
+  'form-control-help-text',
+  'combobox',
+  'display-input',
+  'listbox',
+  'tags',
+  'tag',
+  'tag__base',
+  'tag__content',
+  'tag__remove-button',
+  'tag__remove-button__base',
+  'clear-button',
+  'expand-icon'
+];
+
+export const Parts = {
```

Comment:
I was wondering, if we need the stories for the parts `display-input` and `listbox`. From the description of the parts, these are not visible and thats probably the reason, why I cannot see any red box around it in Storybook.

What was your idea behind it? As a "new" user looking at it, it feels like there is something broken here. All other parts have a red outline but these 2... What do you think?

Reply:
sorry... 

same for `tag__remove-button__base`, `tag__remove-button` & `tag__content`

Reply:
Good point, it isn't my idea so much as Shoelace's.  In my opinion, there are too many parts and they feel redundant like display-input and combobox, but do have different elements in the code.  I just kept the parts Shoelace has.  

Regarding the missing outlines, I don't understand why they aren't visible.  listbox for example has the correct style set but nothing is visible.  border on the is clearly visible:
![image](https://github.com/solid-design-system/solid/assets/39494579/52764fd1-01e9-4763-9728-0e864f7661e9)

I'm not sure how to fix this.  Any ideas @mariohamann ?

Reply:
Ok, the other issues are resolved, this is the last standing topic.  I can take a closer look but I'm not sure what the best solution is here.

Reply:
@abudd1094 pls open up a bug ticket to tackle the issue. I agree to the fact, that we should at least question the parts and/or make them be visible here.